api/envoy/config/listener/v3alpha/listener_components.proto

```diff
@@ -224,4 +264,8 @@ message ListenerFilter {
   oneof config_type {
     google.protobuf.Any typed_config = 3;
   }
+
+  // [#not-implemented-hide:]
+  // Decide when to enable/disable this listener filter on incoming traffic.
```

Comment:
It's not clear to me how this works from documentation, and what the default behavior is if there is no predicate provided. Perhaps this field should be named `enable_match_predicate`? And the default is enabled if no predicate is provided? Can you potentially provide a worked example using a config that would want to disable on MySQL just so we can see what it looks like?

Reply:
I have no opinion on what the name is as we have a `not_match`
`enable_match_predicate` looks good.
Also let me emphasize in comment that if the member is missing it is default enabled as it was.